libs/ui-lib/lib/ocm/components/clusterConfiguration/manifestsConfiguration/components/customManifestsValidationSchema.tsx

```diff
@@ -1,9 +1,13 @@
 import * as Yup from 'yup';
 import { UniqueStringArrayExtractor } from '../../staticIp/commonValidationSchemas';
 import { CustomManifestValues, ManifestFormData } from '../data/dataTypes';
+import { load } from 'js-yaml';
 
+const maxFileSize = 100000; //100 kb
 const FILENAME_REGEX = /^[^/]*\.(yaml|yml|json)$/;
 const INCORRECT_FILENAME = 'Must have a yaml, yml or json extension and can not contain /.';
+const INCORRECT_TYPE_FILE = 'File type is not supported. File type must be yaml, yml or json.';
+const INCORRECT_FILE_SIZE = 'File size is too big. Upload a new 100 Kb or less.';
```

Comment:
```suggestion
const INCORRECT_FILE_SIZE = 'File size is too big. Upload a new 100 Kb or less.';
```

If the file size limit changes you need to update the number in 2 places. Better, make a function out of this message that receives the max file size limit.